entry_types/scrolled/package/src/frontend/MediaPlayer/index.js

```diff
@@ -0,0 +1,28 @@
+import {useEffect, useState, useRef} from 'react';
+import {media} from 'pageflow/frontend';
+
+export function MediaPlayer(props){
+  const playerWrapper = useRef(null);
+  const [player, setPlayer] = useState(undefined);
+  
+  useEffect( () => {    
+    if (props.sources && player == undefined) {
+      setPlayer(media.getPlayer(props.sources, {
+        poster: props.poster
+      }));
+    }
+  } );
```

Comment:
Please install a linter plugin for your editor, so you see React hooks related errors.